packages/fiori/test/specs/Wizard.spec.js

```diff
@@ -35,6 +57,12 @@ describe("Wizard general interaction", () => {
 		assert.strictEqual(step1InHeader.getAttribute("selected"), null,
 			"First step  in the header not is selected.");
 
+		// assert - check if aria-attributes are applied correctly when step is not selected
+		assert.strictEqual(step1InHeaderRoot.getAttribute("role"), "listitem",
+			"First step in the header has role.");
+		assert.strictEqual(step1InHeaderRoot.getAttribute("aria-label"), "Step 1 Product type",
```

Comment:
not a correct way to test those generated texts, please see https://github.com/SAP/ui5-webcomponents/pull/2761/files#diff-41befa9949286af7405d78157c49007ba54c90453e56f9ff1096948248899adc